tests/e2e/e2e_test.go

```diff
@@ -111,3 +109,257 @@ func (s *IntegrationTestSuite) TestBankTokenTransfer() {
 		)
 	})
 }
+
+func (s *IntegrationTestSuite) TestSendTokensFromNewGovAccount() {
+	chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+	senderAddress, err := s.chainA.validators[0].keyInfo.GetAddress()
+	s.Require().NoError(err)
+	sender := senderAddress.String()
+	proposalCounter++
+	s.T().Logf("Proposal number: %d", proposalCounter)
+
+	s.fundCommunityPool(chainAAPIEndpoint, sender)
+
+	s.T().Logf("Submitting Legacy Gov Proposal: Community Spend Funding Gov Module")
+	s.submitLegacyProposalFundGovAccount(chainAAPIEndpoint, sender)
+	s.T().Logf("Depositing Legacy Gov Proposal: Community Spend Funding Gov Module")
+	s.depositGovProposal(chainAAPIEndpoint, sender, proposalCounter)
+	s.T().Logf("Voting Legacy Gov Proposal: Community Spend Funding Gov Module")
+	s.voteGovProposal(chainAAPIEndpoint, sender, proposalCounter, "yes", false)
+
+	initialGovBalance, err := getSpecificBalance(chainAAPIEndpoint, govModuleAddress, photonDenom)
+	s.Require().NoError(err)
+	proposalCounter++
+
+	s.T().Logf("Submitting Gov Proposal: Sending Tokens from Gov Module to Recipient")
+	s.submitNewGovProposal(chainAAPIEndpoint, sender, proposalCounter, "/root/.gaia/config/proposal_2.json")
+	s.T().Logf("Depositing Gov Proposal: Sending Tokens from Gov Module to Recipient")
+	s.depositGovProposal(chainAAPIEndpoint, sender, proposalCounter)
+	s.T().Logf("Voting Gov Proposal: Sending Tokens from Gov Module to Recipient")
+	s.voteGovProposal(chainAAPIEndpoint, sender, proposalCounter, "yes", false)
+	s.Require().Eventually(
+		func() bool {
+			sendGovAmount := sdk.NewInt64Coin(photonDenom, 10)
```

Comment:
should this 10 be a variable so if it changes in one place it updates in both?

Reply:
isn't it included in the gov module send msg as well?

Reply:
in `writeGovProposals ` (https://github.com/cosmos/gaia/pull/1531/files#diff-7d53e76218ae6e6128b32ace0c99cf129d5f294a3aac6a6ddb9f115bc85cece2R457) it has the same 10 photon being sent. Aren't these numbers dependent on each other? If the first one were changed to 5 this test would fail because of an insufficient balance right?

Reply:
Yes you're right, we should pull this out into a single constant